source/performance/F0.Minesweeper.Logic.Benchmarks/Program.cs

```diff
@@ -0,0 +1,20 @@
+using System.Diagnostics;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Running;
+
+IConfig config = Debugger.IsAttached
+	? CreateDebugConfiguration()
+	: CreateBenchmarkConfiguration();
+
+_ = BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args, config);
+
+static IConfig CreateBenchmarkConfiguration()
+	=> DefaultConfig.Instance
+		.WithOptions(ConfigOptions.StopOnFirstError)
+		.AddColumn(StatisticColumn.Min, StatisticColumn.Max, StatisticColumn.Median, RankColumn.Arabic)
+		.AddDiagnoser(MemoryDiagnoser.Default);
+
+static IConfig CreateDebugConfiguration()
+	=> new DebugInProcessConfig();
```

Comment:
Do we want to have the same options, columns and diagnosers on the `DebugInProcessConfig` as for the `DefaultConfig`?

Reply:
I'd say no - because a `[Benchmark]` in "Debug" Configuration - or better to say in non-Release Configuration - doesn't make much sense. And this config does only exist for debugging via `F5`, if we run into problems during the benchmark.